examples/printerdemo_mcu/Cargo.toml

```diff
@@ -15,7 +15,7 @@ path = "main.rs"
 name = "printerdemo_mcu"
 
 [features]
-mcu-simulator = ["slint/renderer-software", "slint/eventloop-winit", "mcu-board-support/std"]
+mcu-simulator = ["slint/renderer-software", "slint/eventloop-winit", "slint/std"]
```

Comment:
unrelated to this patch, but we probably also want to rename this feature to just `simulator` 
But it would be nice if it could somehow enable the sofware renderer in winit automatically.

Reply:
Agreed. This goes along with the documentation overhaul for the MCU bits.